mindbender/plugins/collect_project_editorial_info.py

```diff
@@ -0,0 +1,36 @@
+# collect_project_editorial_info.py
+import pyblish.api
+
+
+class CollectMindbenderEditInfo(pyblish.api.ContextPlugin):
+    """Store projects editorial info at the time of publish"""
+
+    label = "Collect Project Edit Info"
+    order = pyblish.api.CollectorOrder
+    hosts = ["maya"]
+    families = [
+        "mindbender.animation"
+    ]
+
+    def process(self, context):
+        import os
+
+        if "environment" not in context.data:
+            context.data["environment"] = dict()
+
+        prefix = "MINDBENDER_"
+        for key, value in os.environ.items():
+            if not key.startswith(prefix):
+                continue
+
+            key = key[len(prefix):]
+
+            # Convert EDIT_IN to EditIn, FPS to Fps.
+            key = "".join(k.title() for k in key.split("_"))
+
+            # Decorate key with "mindbender" to match prev namning convention
+            decoration = prefix.replace("_", "")
```

Comment:
When in doubt, use examples.

```python
>>> key = "MINDBENDER_EDIT_IN"
>>> "".join(k.title() for k in key.split("_"))
'MindbenderEditIn'
```

Do you want the first letter `M` to be lower-case, so as to match the mixedCase naming convention?

You're right, I did not think of that. Without `MINDBENDER_` it would still not be mixedCase.

```python
>>> key = "EDIT_IN"
>>> "".join(k.title() for k in key.split("_"))
'EditIn'
```

In that case, there isn't to my knowledge any more fancy version than this.

```python
>>> key = "EDIT_IN"
>>> key = "".join(k.title() for k in key.split("_"))
>>> key[0].lower() + key[1:]
'editIn'
```